src/components/Breadcrumbs/Breadcrumbs.tsx

```diff
@@ -43,7 +49,8 @@ const Breadcrumbs: React.FC<BreadcrumbProps> = ({ items, ...rest }) => {
                 data-active={isLastBreadcrumb ? true : undefined}
                 {...rest}
               >
-                {item.text}
+                {/* TODO: remove item.text  */}
```

Comment:
Do we have a target version of `pounce` where we intend to remove `text`? Perhaps we should make an Asana task to migrate `panther-enterprise` and fully remove the value

Reply:
I agree, usually if someone is using this product you shouldn't change properties like this. 
You are doing this in phases!

But you are welcome to provide any thought on this!

Reply:
Yeah totally. I think as long as we add an Asana task to migrate `panther-enterprise` and remove this prop I'm good.

Since we only have 1 app using `pounce` and we're still a `0.x.x` version I'm ok with the breaking change, as long as we provide a migration path.